src/systems/pad.h

```diff
@@ -16,10 +16,21 @@ enum Dir {
     DIR_NONE = -1,
 };
 
+struct StickInputs {
+    s32 x;
+    s32 y;
+};
+
+struct TriggerInputs {
+    s32 l;
+    s32 r;
+};
+
```

Comment:
I try to avoid giving types a plural name so referring to multiple isn't confusing, but not a huge deal. `StickInput` or `StickState` might work